### PR TITLE
Remove redundant loader

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR removes redundant loader that hides the IDE iframe while IDE is loading.

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18734

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
[dashboard-next-ci-test]

**Comment by Ohrimenko1988** [comment]
[dashboard-next-ci-test]

**Review by JPinkney — COMMENTED** [review]
I just have a few minor comments. I tried on che.openshift.io and everything is working perfectly but when I tried locally with my own che instance I got:
```
Refused to frame 'https://keycloak-eclipse-che.apps-crc.testing/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self'".
```
when the workspace is trying to open. If I click away and then re-open the workspace it works though.

workspace-issue

Also, I'm not sure if it's related to this PR but the workspace logs no longer reset after a workspace has been stopped then restarted again

File: webpack.config.common.js [hunk 1]
```diff
@@ -26,38 +26,39 @@ module.exports = {
     path: path.join(__dirname, 'build'),
     publicPath: '/',
     filename: 'client.js',
-    chunkFilename: '[name].[chunkhash].js',
+    // chunkFilename: '[name].[chunkhash].js',
   },
   optimization: {
-    chunkIds: 'named',
```
Comment (olexii4) [inline webpack.config.common.js:32]: We should move it into 'webpack.config.prod.js'.
WDYT?

**Review by olexii4 — APPROVED** [review]
LGTM

File: webpack.config.common.js (hunk 1 above)
Comment (akurinnoy) [reply webpack.config.common.js]: Just didn't revert temporary changes.

File: webpack.config.common.js [hunk 2]
```diff
@@ -26,38 +26,39 @@ module.exports = {
...
-          minChunks: 2,
-          chunks: 'all',
-          priority: 10,
-          reuseExistingChunk: true,
-          enforce: true
-        },
-      },
-    },
+    minimize: false,
+    // chunkIds: 'named',
```
[the remark this answers is not in the document]
Comment (akurinnoy) [reply webpack.config.common.js]: Thanks, reverted temporary changes.

File: src/containers/__tests__/IdeLoader.spec.tsx [hunk 3]
```diff
@@ -100,41 +112,38 @@ describe('IDE Loader container', () => {
...
-          requestWorkspace={async (id: string) => requestWorkspace(id)}
-          startWorkspace={async (id: string) => startWorkspace(id)} />
+        />
       </Provider>,
     );
   };
 
-  beforeEach(() => {
-    showAlert = jest.fn();
+  // beforeEach(() => {
```
[the remark this answers is not in the document]
Comment (akurinnoy) [reply src/containers/__tests__/IdeLoader.spec.tsx]: Thanks, fixed.

[the remark this answers is not in the document]
Comment (akurinnoy) [reply src/containers/__tests__/IdeLoader.spec.tsx]: it is, thanks

**Comment by Ohrimenko1988** [comment]
[dashboard-next-ci-test]

**Review by sleshchenko — APPROVED** [review]
I did not review code carefully but `docker.io/maxura/che-server:che-dashboard-pull-142` seems to work as expected.
I even tried IntelijEditor

<details>
<summary>intelij-java-web-vertx.devfile.yaml</summary>

```yaml
apiVersion: 1.0.0
metadata:
  name: intelij-java-web-vertx
projects:
  - name: vertx-http-example
    source:
      location: 'https://github.com/openshift-vertx-examples/vertx-http-example.git'
      type: git
      branch: master
components:
  - id: che-incubator/intellij-community/latest
    type: cheEditor
  - mountSources: true
    endpoints:
      - name: 8080-tcp
        port: 8080
      - attributes:
          public: 'false'
        name: debug
        port: 5005
    memoryLimit: 512Mi
    type: dockerimage
    volumes:
      - name: m2
        containerPath: /home/user/.m2
    alias: maven
    image: 'quay.io/eclipse/che-java8-maven:nightly'
    env:
      - value: ''
        name: MAVEN_CONFIG
      - value: '-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user'
        name: JAVA_OPTS
      - value: $(JAVA_OPTS)
        name: MAVEN_OPTS
commands:
  - name: maven build
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/vertx-http-example'
        type: exec
        command: 'mvn -Duser.home=${HOME} clean install'
        component: maven
  - name: run app
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/vertx-http-example'
        type: exec
        command: |
          JDBC_URL=jdbc:h2:/tmp/db \
          java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 \
          ./target/*.jar
        component: maven
  - name: stop app
    actions:
      - type: exec
        command: echo 'Stopping the application...' && kill $(pidof java) && echo 'The application has been successfully stopped'
        component: maven
  - name: Debug remote java application
    actions:
      - referenceContent: |
          {
          "version": "0.2.0",
          "configurations": [
            {
              "type": "java",
              "name": "Debug (Attach) - Remote",
              "request": "attach",
              "hostName": "localhost",
              "port": 5005
            }]
          }
        type: vscode-launch
```

</details>

**Comment by Ohrimenko1988** [comment]
[dashboard-next-ci-test]

**Pull request merged**